src/meta/model_v2/migration/src/m20240506_112555_subscription_partial_ckpt.rs

```diff
@@ -0,0 +1,232 @@
+use sea_orm_migration::prelude::{Table as MigrationTable, *};
```

Comment:
Since subscription has not been released yet, it's expected that there is no data in the created tables. Maybe we can just drop all related tables and recreate the new table with the new schema.

Reply:
cc @yezizp2012

Reply:
LGTM